src/Gir/Marshal/Varargs.cs

```diff
@@ -5,7 +5,7 @@ public partial class Varargs : ISymbol
 	{
 		public string CSharpType => "...";
 
-		public string CType => "va_list";
+		public string Name => "va_list";
```

Comment:
Not sure about this one

Reply:
This whole class will be rewritten to support overridden data, so it's fine for now.